launch/cylinder_spawner.launch

```diff
@@ -0,0 +1,5 @@
+<launch>
```

Comment:
if always works well, you should see a force along z in the measure reference frame and with rviz plot on /world_my_sensor/ft_sensor_topic/wrench/force and /world_my_sensor/ft_sensor_topic/wrench/torque topic, you can visualize force and torque, respectively.